config/initializers/hets_agent_initializer.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# A service class that defines and sends the Hets version requirement.
+class HetsAgentIninializer
+  VERSION_REQUIREMENT = '>= 0.100.0'
```

Comment:
Can we read this from the settings file? Seems easier to find. Or at least somewhere more central (`application.rb`?)

Reply:
I would like to keep it away from the settings file because it is related directly to the codebase and not part of what an admin shall change.

Reply:
Yeah, but I wouldn't hide it away in an initializer file either, since this is something that might need to be changed from time to time.

Reply:
Fixed it (moved the version requirement to the config/application.rb) by commit amend.